.github/workflows/release-automation.yml

```diff
@@ -183,12 +183,19 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@6d6857d36972b65feb161a90e484f2984215f83e # v6.0.5
         with:
-          body: ''
+          body: |
+            **Description**:
+            Bump versions for v${{ env.NEXT_VERSION_SNAPSHOT }}
+            Automated snapshot version bump for the next development cycle.
+
+            **Related issue(s)**:
```

Comment:
Are there expected to be `Related Issue(s)` populating here?

Reply:
Ah, yes, I intended for it to be linked to the release ticket for each corresponding release. For example, this [v0.64.0-SNAPSHOT PR](https://github.com/hashgraph/hedera-json-rpc-relay/pull/3355) is linked to the [release/0.63 ticket](https://github.com/hashgraph/hedera-json-rpc-relay/issues/3354)